services/gitlab/gitlab-release.service.js

```diff
@@ -0,0 +1,136 @@
+import Joi from 'joi'
+import { optionalUrl } from '../validators.js'
+import { latest, renderVersionBadge } from '../version.js'
+import { NotFound } from '../index.js'
+import GitLabBase from './gitlab-base.js'
+
+const schema = Joi.array().items(
+  Joi.object({
+    name: Joi.string().required(),
+    tag_name: Joi.string().required(),
+  })
+)
+
+const queryParamSchema = Joi.object({
+  gitlab_url: optionalUrl,
+  include_prereleases: Joi.equal(''),
+  sort: Joi.string().valid('date', 'semver').default('date'),
+  display_name: Joi.string().valid('tag', 'release').default('tag'),
```

Comment:
Figured it made sense to add this from day 0 given where we ended up with this on GitHub. I've opted to set the default to `tag` to be consistent with what GitHub currently is but don't have strong opinions on this one way or another.